src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy

```diff
@@ -298,6 +307,46 @@ class FlowHistoryV2Spec extends HealthCheckSpecification {
         ]
     }
 
+    @Tidy
+    @Tags([VIRTUAL, LOW_PRIORITY])
+    //it doesn't work on the hardware env due to the 'round trip latency' feature
```

Comment:
It will work as it is now, but it's only because of our current setup.
1. You can explicitly pick switches that do not support rtl, so that this behavior is strictly defined
2. or You can force a flow reroute by bringing some port down on the path. This way you will no longer be dependent on whether isls are rtl or not

Reply:
agree, refactored.
the first option was implemented